client/src/views/SpecificJobPage/SpecificJobPage.tsx

```diff
@@ -0,0 +1,314 @@
+      color: 'white',
+    },
+  })
+  const StyledShortlistButton = styled(MUIButton)({
+    '&.MuiButton-textPrimary': {
+      color: '#0082D5',
+      width: '100%',
+      cursor: 'pointer',
+      border: '1px solid #058DDA',
+    },
+  })
+  const JobBody = () => (
+    <div>
+      {imageURL !== '' && (
+        <Tabs
+          TabIndicatorProps={{ style: { backgroundColor: 'white' } }}
+          value={tabSelected}
+          onChange={handleTabChange}
+          sx={{ marginLeft: '150px', paddingTop: '16px' }}
+        >
+          <StyledTab label="Details" />
+          {/* <StyledTab label="Reviews" /> */}
+        </Tabs>
+      )}
+      {imageURL !== '' && (
+        <Box sx={{ display: tabSelected === 0 ? 'flex' : 'none' }}>
+          <Description />
+          <Box
+            bgcolor="white"
+            sx={{
+              alignSelf: 'flex-start',
+              minWidth: '15vw',
+              maxWidth: '20%',
+              borderRadius: '16px',
+              p: 2,
+              m: 4,
+            }}
+          >
+            <Button
+              sx={{ borderRadius: '50px', marginBottom: '10px' }}
+              className="BlueGradientBackground"
+              width="100%"
+              href={`https://waterlooworks.uwaterloo.ca/myAccount/co-op/coop-postings.htm?ck_jobid=${jobId}`}
+              target="_blank"
+            >
+              Open in WaterlooWorks
+            </Button>
+            {/* <StyledShortlistButton
+              sx={{ borderRadius: '50px', marginBottom: '10px' }}
+              href={`https://waterlooworks.uwaterloo.ca/myAccount/co-op/coop-postings.htm?ck_jobid=${jobId}`}
+              target="_blank"
+            >
+              Shortlist
+            </StyledShortlistButton> */}
+            {extraInfo &&
+              extraInfo.map((item: { title: string; text: string }) => {
+                return (
+                  item.text && (
+                    <>
+                      <Typography variant="h6" fontWeight="bold">
+                        {item.title}
+                      </Typography>
+                      <Typography>{ReactHtmlParser(item.text)}</Typography>
+                    </>
+                  )
+                )
+              })}
+            <Typography variant="h6" fontWeight="bold" />
+          </Box>
+        </Box>
+      )}
+      {/* <Box sx={{ display: tabSelected === 1 ? 'flex' : 'none' }}>bonjour</Box> */}
+    </div>
+  )
+
+  return (
+    <>
+      <NavigationBar />
+      <Container style={{ paddingLeft: '170px' }}>
+        <MainWrapper>
+          <Spacer height={48} />
+          <CompanyHeader />
+          <Spacer height={32} />
+        </MainWrapper>
+      </Container>
+      <WaterWrapper>
+        <Container>
+          <MainWrapper>
+            <JobBody />
+          </MainWrapper>
+        </Container>
+      </WaterWrapper>
+    </>
+  )
+}
+
+const MainWrapper = styled.div`
+  display: flex;
+  align-items: left;
```

Comment:
random q:
what does align-items left do?

was tryna search it up but idt it exists?

Reply:
it aligns all the components in the wrapper to the left

Reply:
shouldn't it be `flex-start`?
idk im a css noob but I never heard of `left`

Reply:
lmaoo mb i'll change that i didnt really goggle it